File: CodeWizards/Model/CircularUnit.cs
using System;

namespace Com.CodeGame.CodeWizards2016.DevKit.CSharpCgdk.Model {
    public abstract class CircularUnit : Unit {
        private readonly double radius;

        protected CircularUnit(long id, double x, double y, double speedX, double speedY, double angle, Faction faction,
                double radius)
                : base(id, x, y, speedX, speedY, angle, faction) {
            this.radius = radius;
        }

        public double Radius => radius;
    }
}

File: CodeWars/local_runner/plugins/model/CircularUnit.java
package model;

@SuppressWarnings({"AbstractClassWithoutAbstractMethods", "AbstractClassNeverImplemented"})
public abstract class CircularUnit extends Unit {
    private final double radius;

    protected CircularUnit(long id, double x, double y, double radius) {
        super(id, x, y);

        this.radius = radius;
    }

    public double getRadius() {
        return radius;
    }
}
